web/libs/core/src/pages/AccountSettings/AccountSettings.tsx

```diff
@@ -0,0 +1,72 @@
+import { Card, CardHeader, CardTitle, CardContent, CardDescription } from "@humansignal/ui/lib/card-new/card";
+import { useMemo } from "react";
+import { Redirect } from "react-router-dom";
+import styles from "./AccountSettings.module.scss";
+import { accountSettingsSections } from "./sections";
+import { SidebarMenu } from "/apps/labelstudio/src/components/SidebarMenu/SidebarMenu";
```

Comment:
We shouldn't do this, it is a cyclic import across packages, importing an app based thing in the package that later uses this package in the app.